packaging/linux/run_keybase

```diff
@@ -45,7 +45,7 @@ echo Launching keybase service...
 keybase service &>> "$logdir/keybase.service.log" &
 echo Mounting /keybase...
 kbfsfuse -debug -mdserver mdserver.kbfs.keybase.io:443 \
-  -bserver bserver.kbfs.keybase.io:443 /keybase &>> "$logdir/keybase.kbfs.log" &
+  -bserver bserver.kbfs.keybase.io:443 -log-to-file /keybase &>> "/dev/null" &
```

Comment:
@oconnor663 We can't `log send` without explicit permission, because it tells us which files a user was accessing.